nss-day-cohort-45/tabloid-cli-cheerseses
# Note Management

# Note Management

## List, Add, and Remove Notes


### How To Test
1. Press 4 for Post Management
2. Press 6 for Note Management
3. See if you can list notes
4. See if you can add notes (then list to see if it works)
5. See if you can delete notes (then list to double check)
6. Press 0 for return to go back
7. Then hit 5 for Post Details
8. Choose a post
9. Then see if you can access Note Management by pressing 4
